### ISSUE TITLE
Poor linting of nested properties in SCSS

### ISSUE THREAD

**Issue by clarcharr** [stylelint/stylelint#2642]
The below code messes with the parser:

```
margin: auto {
  top: $left;
  bottom: $right;
}
```

This is valid in SCSS, so, the parser shouldn't emit parsing errors here.

**Comment by ntwb** [comment]
You'll need to use [stylelint-scss](https://github.com/kristerkari/stylelint-scss)

Also GitHub issues are for stylelint bugs and enhancements, if you're looking for help or support with stylelint then stackoverflow is our preferred question and answer forum - http://stackoverflow.com/questions/tagged/stylelint
